tests/load/test_job_client.py

```diff
@@ -719,15 +719,12 @@ def test_retrieve_job(client: SqlJobClientBase, file_storage: FileStorage) -> No
     }
     with io.BytesIO() as f:
         write_dataset(client, f, [load_json], client.schema.get_table(user_table_name)["columns"])
-        dataset = f.getvalue().decode()
-    job = expect_load_file(client, file_storage, dataset, user_table_name)
+        # dataset = f.getvalue().decode()
+    # job = expect_load_file(client, file_storage, dataset, user_table_name)
     # now try to retrieve the job
     # TODO: we should re-create client instance as this call is intended to be run after some disruption ie. stopped loader process
-    r_job = client.restore_file_load(file_storage.make_full_path(job.file_name()))
-    assert r_job.state() == "completed"
-    # use just file name to restore
-    r_job = client.restore_file_load(job.file_name())
-    assert r_job.state() == "completed"
+    # r_job = client.restore_file_load(file_storage.make_full_path(job.file_name()))
```

Comment:
NOTE: todo

Reply:
for bigquery restoring is tested now, the other just re-run the job when found in the running folder (which as discussed is the right way to do it)